docs/content/users/basics/commands.md

```diff
@@ -0,0 +1,1230 @@
+# Commands
+
+You can tell DDEV what to do by running its commands. This page details each of the available commands and their options, or flags.
+
+Run DDEV without any commands or flags to see this list in your terminal:
+
+```
+→  ddev
+Create and maintain a local web development environment.
+Docs: https://ddev.readthedocs.io
+Support: https://ddev.readthedocs.io/en/stable/users/support
+
+Usage:
+  ddev [command]
+
+Available Commands:
+  auth             A collection of authentication commands
+  blackfire        Enable or disable blackfire.io profiling (global shell web container command)
+  clean            Removes items ddev has created
+  composer         Executes a composer command within the web container
+...
+```
+
+Use [`ddev help`](#help) to learn more about a specific command, like this example for [`ddev describe`](#describe):
+
+```
+→  ddev help describe
+Get a detailed description of a running ddev project. Describe provides basic
+information about a ddev project, including its name, location, url, and status.
+It also provides details for MySQL connections, and connection information for
+additional services like MailHog and phpMyAdmin. You can run 'ddev describe' from
+a project directory to describe that project, or you can specify a project to describe by
+running 'ddev describe <projectname>'.
+
+Usage:
+  ddev describe [projectname] [flags]
+
+Aliases:
+  describe, status, st, desc
+
+Examples:
+ddev describe
+ddev describe <projectname>
+ddev status
+ddev st
+
+Flags:
+  -h, --help   help for describe
+
+Global Flags:
+  -j, --json-output   If true, user-oriented output will be in JSON format.
+```
+
+## Global Flags
+
+Two flags are available for every command:
+
+* `--help` or `-h`: Outputs more information about a command rather than executing it.
+* `--json-output` or `-j`: Format user-oriented output in JSON.
+
+---
+
+## `auth`
+
+A collection of authentication commands.
```

Comment:
Since there's no longer a collection... (there once was)... should we leave this one out?

Reply:
Leaving it out feels like cheating since it exists. But we could surely align the description more closely with reality:

```suggestion
## `auth`

Authentication commands.
```

Reply:
Works for me.